src/julia-parser.scm

```diff
@@ -518,6 +518,8 @@
       (and (char>=? c #\u200c) (char<=? c #\u200f))
       (memv c '(#\u00ad #\u2061 #\u115f))))
 
+(define (scolno port) (string " near column " (input-port-column port)))
```

Comment:
Can we have line number too: " near line 2 column 3" (`"near line " (input-port-line port) " column " (input-port-column port)`)?

Reply:
Don't we already get a line number from the `LoadError`?

Reply:
For example:
```
julia> include_string(Main, string(Char(0xdcdb)," = 1"))
ERROR: LoadError: syntax: invalid character "???" near column 0
Stacktrace:
 [1] include_string(::Module, ::String, ::String) at ./loading.jl:1008
 [2] include_string(::Module, ::String) at ./loading.jl:1012
 [3] top-level scope at none:0
in expression starting at string:1
```
The line number is already there (`at string:1`).

Reply:
Ah, ok. I see that was added into https://github.com/JuliaLang/julia/pull/16748

Although that also only applies to cases where it is using `include`. Direct uses of the parser still would need to handle that for themselves:
```
julia> Meta.parse(string("\n\n\n", Char(0xdcdb), " = 1"), 1, raise=false)
($(Expr(:error, "invalid character \"\udcdb\""), 7)
```
Although I guess those cases might be better served by CSTParser.jl anyways.